src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Automation.Peers/PersonPictureAutomationPeer.cs

```diff
@@ -9,7 +9,9 @@ public  partial class PersonPictureAutomationPeer : global::Windows.UI.Xaml.Auto
 	{
 		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
 		[global::Uno.NotImplemented("__ANDROID__", "__IOS__", "NET461", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
-		public PersonPictureAutomationPeer( global::Windows.UI.Xaml.Controls.PersonPicture owner) : base(owner)
+		/* Manually adjusted parameter type. Otherwise, we get obsoletion warning. */
+		/* TODO: Adjust sync generator to always generate this with MUXC instead of WUXC */
```

Comment:
I guess my change here is wrong. I should be leaving this and creating a parallel MUXC type.